spec/support/shared_examples_for_providers.rb

```diff
@@ -32,7 +32,10 @@
 shared_examples_for 'Rollbar integrated error handler' do
   context 'uncaught in a controller' do
     it 'reports the error to Rollbar' do
-      Rollbar.should_receive(:log).with('error', instance_of(RuntimeError)).and_return({})
+      Rollbar.should_receive(:log) do |level, args|
+        expect(level).to eq('error')
```

Comment:
sweet jesus it's john

On Tue, Feb 24, 2015 at 4:05 PM, John Wilkinson <notifications@github.com>
wrote:

> In spec/support/shared_examples_for_providers.rb
> <https://github.com/lumoslabs/smoke_detector/pull/16#discussion_r25306871>
> :
>
> > @@ -32,7 +32,10 @@
> >  shared_examples_for 'Rollbar integrated error handler' do
> >    context 'uncaught in a controller' do
> >      it 'reports the error to Rollbar' do
> > -      Rollbar.should_receive(:log).with('error', instance_of(RuntimeError)).and_return({})
> > +      Rollbar.should_receive(:log) do |level, args|
> > +        expect(level).to eq('error')
>
> double quotes suck
>
> FTFY
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lumoslabs/smoke_detector/pull/16/files#r25306871>.
>